mirage/config.js

```diff
@@ -6,6 +6,9 @@ export default function () {
   this.urlPrefix = okapi.url;
   this.get('/_/version', () => '0.0.0');
   this.get('/_/proxy/modules', []);
+  this.get('/saml/check', {
+    ssoEnabled: false
```

Comment:
I should note that I'm not actually sure if that's the correct response for that endpoint, but it seems to clear up the errors that were being thrown without it.